docs/user/registering-microshift-devices-acm.md

```diff
@@ -0,0 +1,157 @@
+
+As part of this setup, you will need to collect the following information:
+
+- Agent registration URL
+- Token
+
+## Creating Repositories in Flight Control
+
+Flight Control includes a configuration provider that can retrieve data from an HTTP(S) endpoint. This integrates with ACM's agent registration API, requiring it to be formatted as a Flight Control repository.
+
+A sample YAML file for an HTTP repository looks like:
+
+```console
+apiVersion: v1alpha1
+kind: Repository
+metadata:
+  labels: {}
+  name: acm-test
+spec:
+  httpConfig:
+    token: $token
+  type: http
+  url: https://$agent_registration_host
+```
+
+Note that this repository will show as not valid, because the root URL does not return a value directly. However, this endpoint will be used in the fleet spec configuration section with a set of suffixes that will return the required Kubernetes manifests for auto-registration.
+
+## Fleet Definition Overview
+
+The following fleet definition provides an example of how devices running MicroShift can be auto-registered into an ACM hub. Here is a breakdown of its sections:
+
+```console
+apiVersion: v1alpha1
+kind: Fleet
+metadata:
+  labels: {}
+  name: fleet-acm
+spec:
+  selector:
+    matchLabels:
+      fleet: acm
+  template:
+    metadata:
+      generation: 1
+      labels:
+        fleet: acm-test
+    spec:
+      os:
+        image: quay.io/myorg/device-image-with-microshift:v1
+      config:
+      - configType: HttpConfigProviderSpec
+        httpRef:
+          filePath: /var/local/crd.yaml
+          repository: acm-test
+          suffix: /agent-registration/crds/v1
+        name: acm-crd
+      - configType: HttpConfigProviderSpec
+        httpRef:
+          filePath: /var/local/import.yaml
+          repository: acm-test
+          suffix: /agent-registration/manifests/{{ device.metadata.name }}
+        name: acm-import
+      hooks:
+        afterUpdating:
+          - path: "/var/local/crd.yaml"
+            onFile: [Create]
+            actions:
+              - executable:
+                  run: "kubectl apply -f /var/local/crd.yaml"
+                  envVars: ["KUBECONFIG=/var/lib/microshift/resources/kubeadmin/kubeconfig"]
+                  workDir: "/usr/bin/"
+          - path: "/var/local/import.yaml"
+            onFile: [Create]
+            actions:
+              - executable:
+                  run: "kubectl apply -f /var/local/import.yaml"
+                  envVars: ["KUBECONFIG=/var/lib/microshift/resources/kubeadmin/kubeconfig"]
+                  workDir: "/usr/bin/"
+```
+
+### Fleet Specification Breakdown
+
+The fleet specification is composed of three main sections: OS, config and hooks.
+
+1. **Operating System (OS) Image**: This section defines the target Operating System image required for all devices, in this case an image containing the MicroShift runtime. This is essential for running the Klusterlet, the agent responsible for communication with ACM.
+
```

Comment:
I'd just make this a pre-requisite at the beginning, so you don't need to repeat explanations. Like "Prerequisite: * You have a fleet of devices running an OS image containing MicroShift."

Reply:
+1